SSPSolution/GraphicsDLL/GraphicsHandler.cpp

```diff
@@ -294,17 +408,23 @@ int GraphicsHandler::Render(float deltaTime)
 
 	m_shaderControl->SetActive(ShaderControl::Shaders::DEFERRED);
 	m_shaderControl->SetVariation(ShaderLib::ShaderVariations::Normal);
-
+	Resources::Model* modelPtr = nullptr;
```

Comment:
This was removed in auto-merge. Is it suppose to be gone? Functions lower is dependent on this pointer.